src/main/java/nl/tudelft/contextproject/camera/Camera.java

```diff
@@ -331,18 +367,16 @@ public int getPresetAmount() {
     }
 
     @Override
-    public boolean equals(Object o) {
-        if (this == o) {
-            return true;
-        }
-        if (!(o instanceof Camera)) {
-            return false;
+    public boolean equals(Object obj) {
+        if (obj instanceof Camera && obj != null) {
```

Comment:
Redundant null check. If `obj instanceof Camera` returns true, then `obj` is automatically not null, because it's apparently an instance of Camera.